### PR TITLE
replace instance variable with local variable

### PR DESCRIPTION
**Pull request by gmichokostas**
Just a small commit that removes some warnings from rails_best_practices

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -383,7 +383,7 @@ GEM
     net-ssh (2.9.1)
     netrc (0.10.2)
     newrelic_rpm (3.9.9.275)
-    nio4r (1.0.1)
+    nio4r (1.1.1)
```
Comment (gmichokostas) [inline Gemfile.lock]: Unfortunately I had to upgrade to make it work on my PC.

Comment (marcaltmann) [inline Gemfile.lock]: Thanks for the PR. Would it be better to upgrade the whole eye gem instead?

Comment (gmichokostas) [inline Gemfile.lock]: Hi @marcaltmann, thanks for your note. I will upgrade the eye gem then. Shall I squash the commits also?

Comment (marcaltmann) [inline Gemfile.lock]: Yes, please.

**Pull request closed without merging**